app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_comments_counter
```

Comment:
* Kindly Set up associations between Your models as shown in the[ ERD diagram ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) correctly, as you can see from the ERD diagram the association between the user and comments table is  `Author_id`  kindly setup the association correctly in all your models 🙏